packages/flutter_tools/bin/macos_assemble.sh

```diff
@@ -79,19 +79,19 @@ fi
 
 RunCommand "${FLUTTER_ROOT}/bin/flutter"                                    \
     ${verbose_flag}                                                         \
-    ${flutter_engine_flag}                                                  \
-    ${local_engine_flag}                                                    \
+    "${flutter_engine_flag}"                                                \
```

Comment:
Question: why not on ${verbose_flag}" above? IIRC none of these variables should contain spaces, but shellcheck won't know that. I'm surprised it didn't comment on that one.

Reply:
Maybe it knows it is either set to `""` or `"--verbose"`?